### PR TITLE
some debug statements and bumping up alpakka

### LINKED ISSUE #3735: Bump up Alpakka release
Bump up alpakka to remove release candidate

### PR DESCRIPTION
**Pull request by omnipresent07**
closes #3735

### CODE REVIEW

File: data-browser/src/main/scala/hmda/dataBrowser/services/S3FileService.scala [hunk 1]
```diff
@@ -26,6 +26,8 @@ class S3FileService(implicit mat: Materializer) extends FileService with Setting
       val content = formName(queries)
       s"${content}${fileEnding(delimiter)}"
     }
+    log.info("Persisting S3 Key: " + key)
```
Comment (PatrickGoRaft) [inline data-browser/src/main/scala/hmda/dataBrowser/services/S3FileService.scala:29]: Can you remove these debugging statements @omnipresent07

Comment (omnipresent07) [reply data-browser/src/main/scala/hmda/dataBrowser/services/S3FileService.scala]: @PatrickGoRaft Changed to debug statements

File: data-browser/src/main/scala/hmda/dataBrowser/services/S3FileService.scala [hunk 2]
```diff
@@ -51,6 +53,8 @@ class S3FileService(implicit mat: Materializer) extends FileService with Setting
 
   override def retrieveData(queries: List[QueryField], delimiter: Delimiter, year: String): Task[Option[Source[ByteString, NotUsed]]] = {
     val key = s3Key(queries, delimiter, year)
+    log.info("Retrieveing S3 Key: " + key)
```
Comment (PatrickGoRaft) [inline data-browser/src/main/scala/hmda/dataBrowser/services/S3FileService.scala:56]: @omnipresent07  can you change these to debugging as well

File: data-browser/src/main/scala/hmda/dataBrowser/api/DataBrowserHttpApi.scala [hunk 3]
```diff
@@ -41,6 +41,7 @@ private class DataBrowserHttpApi(log: Logger, fileCache: S3FileService, query: Q
         )
 
       case Success(Right(url)) =>
+        log.info("This is the URL for S3: " + url)
```
Comment (PatrickGoRaft) [inline data-browser/src/main/scala/hmda/dataBrowser/api/DataBrowserHttpApi.scala:44]: @omnipresent07  here as well

**Pull request merged**